utils/sql/db_update_manifest.txt

```diff
@@ -443,6 +443,7 @@
 9187|2022_07_09_task_zone_version_matching.sql|SHOW COLUMNS FROM `task_activities` LIKE 'zone_version'|empty|
 9188|2022_07_14_zone_expansion_revert.sql|SHOW COLUMNS FROM `zone` LIKE 'expansion'|empty|
 9189|2022_07_10_character_task_rewarded.sql|SHOW COLUMNS FROM `character_tasks` LIKE 'was_rewarded'|empty|
+9190|2022_07_18_rule_death_keep_level.sql|SELECT * FROM `rule_values` WHERE `rule_name` LIKE '%Character:DeathKeepLevel%'|empty|
```

Comment:
Rules are automatically inserted, this is unneeded, should be removed

Reply:
@mackal , I also removed the migration file, https://github.com/EQEmu/Server/pull/2319/commits/de93b545fab621623aa3d97aba7f98481404ecd9, but should I have kept that in?

I'm trying to test locally but not seeing the rule added in the database, but very well am doing something else wrong in getting this set up.